packages/babel-preset-react-app/index.js

```diff
@@ -115,6 +115,16 @@ module.exports = function(api, opts) {
           regenerator: true,
         },
       ],
+      !isEnvTest && [
+        require('babel-plugin-named-asset-import'),
```

Comment:
What do you think about moving this to webpack configs instead of keeping it in the preset? That would make more sense to me.

Reply:
I know we talked about that before and I tried it but I couldn't get it to work. For some reason when I move this to the webpack config it strips out functions. In this case the `svg` key would be present in the `loaderMap` object but it would just be an empty object. So the `ReactComponent` key would be removed presumably because its value is a function.

Reply:
Can you push a commit that tries to implement it, even if it doesn’t work? I think we need to figure out what breaks there. Fine to do as another PR against your existing PR if you prefer so

Reply:
Actually try rebasing. We don’t write babelrc anymore (afaik) since monorepos landed. So it doesn’t get serialized on eject and thus it may be solved now

Reply:
I merged the `next` branch (with the monorepo stuff) into this branch a couple of commits ago. That should have the same effect right? You're saying I should try moving this to the webpack config again and see if it works now?

Reply:
Yes